ironfish/src/rpc/routes/wallet/createTransaction.ts

```diff
@@ -0,0 +1,225 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Asset } from '@ironfish/rust-nodejs'
+import { BufferMap } from 'buffer-map'
+import * as yup from 'yup'
+import { BurnDescription } from '../../../primitives/burnDescription'
+import { MintDescription } from '../../../primitives/mintDescription'
+import { RawTransactionSerde } from '../../../primitives/rawTransaction'
+import { CurrencyUtils } from '../../../utils'
+import { NotEnoughFundsError } from '../../../wallet/errors'
+import { ERROR_CODES, ValidationError } from '../../adapters/errors'
+import { ApiNamespace, router } from '../router'
+
+export type CreateTransactionRequest = {
+  fromAccountName: string
```

Comment:
This has 2 different names, `sender` in the `wallet.createTransaction()`, `fromAccountName` here, and `account` in other RPCs. Can we choose one?